ui/src/data/components/__tests__/DataItem.test.jsx

```diff
@@ -19,51 +20,59 @@ describe('DataItem', () => {
             {
               curated_relation: true,
               record: {
-                $ref: "http://M1/api/institutions/12346"
+                $ref: 'http://M1/api/institutions/12346',
               },
-              value: "ut"
+              value: 'ut',
             },
-          ]
-        }
+          ],
+        },
       ],
       urls: [{ value: 'http://url.com' }],
       dois: [
         {
-          source: "in ad et",
-          value: "10.8756/tTM",
-          material: "data"
+          source: 'in ad et',
+          value: '10.8756/tTM',
+          material: 'data',
         },
         {
-          source: "mollit deserunt eu",
-          value: "10.5/.Aww=bT@",
-          material: "version"
+          source: 'mollit deserunt eu',
+          value: '10.5/.Aww=bT@',
+          material: 'version',
         },
         {
-          source: "adipisicing et",
-          value: "10.0.9747720/#}O=W:$",
-          material: "part"
-        }
+          source: 'adipisicing et',
+          value: '10.0.9747720/#}O=W:$',
+          material: 'part',
+        },
       ],
-      control_number: 1234
+      control_number: 1234,
     });
 
-    const wrapper = shallow(<DataItem metadata={metadata} />);
-    expect(wrapper).toMatchSnapshot();
+    const { asFragment } = render(
+      <MemoryRouter initialEntries={['/data/123']} initialIndex={0}>
```

Comment:
here it should be /data and not /data/123 as the test is related to DataItem (part of the Search)